src/experiments.js

```diff
@@ -37,6 +42,9 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 /** @type {Object<string, boolean>} */
 let toggles_ = null;
 
+/** @type {!Promise} */
+const originTrialsPromise = enableExperimentsForOriginTrials(self);
```

Comment:
But how can `chunk` control when the different parts of the task run if the task has a long promise chain?